roofit/roofitcore/src/RooHist.cxx

```diff
@@ -87,6 +87,11 @@ RooHist::RooHist(const TH1 &data, Double_t nominalBinWidth, Double_t nSigma, Roo
        Bool_t correctForBinWidth, Double_t scaleFactor) :
   TGraphAsymmErrors(), _nominalBinWidth(nominalBinWidth), _nSigma(nSigma), _rawEntries(-1)
 {
```

Comment:
I don't understand why we don't support this.
I think one can have both case, and being able to create a RooHist also without any bin width correction, depending on the passed option